### PR TITLE
Taras.krasitskyi.selenide

### PR DESCRIPTION
**Pull request by Taras-Kr**

### CODE REVIEW

File: src/test/java/com/ss/ita/google/SearchLinksContainsWordTest.java [hunk 1]
```diff
@@ -20,4 +23,17 @@ public void verifySearchLinksContainsWord() {
...
         }
     }
+
+    @Test
+    public void verifyFivePageLinksContainsText() {
+        String searchText = "funny kitten";
+        int numberResultPage = 5;
+        List<String> linksTexts = new HomePage().doSearch(searchText).openResultPage(5).getResultLinksTexts();
+        String expectedText = "kitten";
+        SoftAssert softAssert = new SoftAssert();
```
Comment (vladdmytriv) [inline src/test/java/com/ss/ita/google/SearchLinksContainsWordTest.java:33]: why U create softassert if U use it only once?

**Review by vladdmytriv — CHANGES_REQUESTED** [review]
SoftAssert?

**Review by vladdmytriv — APPROVED** [review]

File: src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java [hunk 2]
```diff
@@ -46,12 +46,12 @@ public HomePage goBackToHomePage() {
...
 
     public SearchedPage openSearchResultLink(int numberResultLink) {
-        getSearchedResultLink(numberResultLink).click();
+        $x(String.format("(//div[@class='yuRUbf']/a)[%s]", numberResultLink)).click();
         return new SearchedPage();
     }
 
     public String getResultLinkUrl(int numberResultLink) {
-        return getSearchedResultLink(numberResultLink).getAttribute("href");
+        return $x(String.format("(//div[@class='yuRUbf']/a)[%s]", numberResultLink)).getAttribute("href");
```
Comment (Percifalll) [inline src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java:54]: ur duplicating web elements searching
move it to general method

File: src/test/java/com/ss/ita/google/SearchLinksContainsWordTest.java [hunk 3]
```diff
@@ -20,4 +23,17 @@ public void verifySearchLinksContainsWord() {
             assertTrue(resultLink.contains(word));
         }
     }
+
+    @Test
+    public void verifyFivePageLinksContainsText() {
+        String searchText = "funny kitten";
+        int numberResultPage = 5;
+        List<String> linksTexts = new HomePage().doSearch(searchText).openResultPage(5).getResultLinksTexts();
```
Comment (Percifalll) [inline src/test/java/com/ss/ita/google/SearchLinksContainsWordTest.java:31]: when ur using method chaining, start each method call from a new line

**Review by Percifalll — CHANGES_REQUESTED** [review]
method chaining
duplicating web-element search

**Pull request merged**